hack/update-protobuf.sh

```diff
@@ -46,4 +46,4 @@ go-to-protobuf \
   --packages="$(IFS=, ; echo "${PACKAGES[*]}")" \
   --apimachinery-packages='-k8s.io/apimachinery/pkg/util/intstr,-k8s.io/apimachinery/pkg/api/resource,-k8s.io/apimachinery/pkg/runtime/schema,-k8s.io/apimachinery/pkg/runtime,-k8s.io/apimachinery/pkg/apis/meta/v1,-k8s.io/apimachinery/pkg/apis/meta/v1beta1,-k8s.io/api/core/v1,-k8s.io/api/rbac/v1,-k8s.io/api/autoscaling/v1,-k8s.io/api/networking/v1' \
   --go-header-file=${PROJECT_ROOT}/hack/LICENSE_BOILERPLATE.txt \
-  --proto-import=${PROJECT_ROOT}/vendor
+#  --proto-import=${PROJECT_ROOT}/vendor
```

Comment:
What does this `-proto-import` flag do, don't we not need it anymore?

Reply:
What is not needed as we rely now on the GOPATH to contain the k8s protobuf files. As the `--proto-import` defaults to the correct location, we can remove it: https://github.com/kubernetes/code-generator/blob/master/cmd/go-to-protobuf/protobuf/cmd.go#L92